management-interface.go

```diff
@@ -320,9 +318,9 @@ func deleteNetwork(id string) error {
 
 	//check if is bushnet
 	var ssid string
-	ssid, err = getNetworkSsid(id)
+	ssid, err = getNetworkSSID(id)
 	if strings.ToLower(ssid) == "\"bushnet\"" {
-		log.Printf("error bushnet cannot be deleted")
+		err = errors.New(fmt.Sprintf("error bushnet cannot be deleted"))
 		return err
```

Comment:
There's no need to assign to err if you're just returning anyway. Also, the use of fmt.Sprintf is unnecessary here. This would be fine:
```go
return errors.New("error bushnet cannot be deleted")
```